features/bookmarks/bookmark_privacy.feature

```diff
@@ -127,3 +118,14 @@ Feature: Private bookmarks
       And I should see "Public Masterpiece"
       And I should see "otheruser"
       And I should not see "bookmarker"
+
+    # Private bookmarks should not show on tag's page
+    When I view the tag "Stargate SG-1"
```

Comment:
I think you can do `When I go to the bookmarks tagged "Stargate SG-1"` here, if you'd like to save a step.